workflow/scripts/analysis/predictive_signature_testing.py

```
@@ -0,0 +1,161 @@
+                                        outcome_column_label=OUTCOME_VARIABLES["event_label"],
+                                        standard_column_label="survival_event_binary",
+                                        event_column_value_mapping=OUTCOME_VARIABLES["event_value_mapping"],
+                                        )
+clinical_data = timeOutcomeColumnSetup(dataframe_with_outcome=clinical_data,
+                                       outcome_column_label=OUTCOME_VARIABLES["time_label"],
+                                       standard_column_label="survival_time_years",
+                                       convert_to_years=OUTCOME_VARIABLES["convert_to_years"])
+
+
+performance_results = list()
+image_types = {str(type.name).removesuffix("_features.csv") for type in sorted(RESULTS_DATA_PATH.rglob("**/*_features.csv"))}
```

Ruff (0.8.2)

Variable `type` is shadowing a Python builtin

(A001)